tempest/api/workloadmgr/cli/test_wlm_cli.py

```diff
@@ -551,3 +551,85 @@ def test_10_token(self):
+            # Create workload with CLI command
+            workload_create = command_argument_string.workload_create + \
+                " instance-id=" + str(self.vm_id)
+            rc = cli_parser.cli_returncode(workload_create)
+            if rc != 0:
+                reporting.add_test_step(
+                    "Execute workload-create command", tvaultconf.FAIL)
+                raise Exception("Command did not execute correctly")
+            else:
+                reporting.add_test_step(
+                    "Execute workload-create command", tvaultconf.PASS)
+                LOG.debug("Command executed correctly")
+            time.sleep(10)
+             
+            # List VMs 
+            vm_list = self.list_vms()
+            LOG.debug("\nVM lists are : {}\n".format(str(vm_list)))
+
+            #Fetch protected VMs : First We need to fetch the instances from WL list and then remove them who are not in the vm_list so WL VMs who are deleted from OpenStack but they still part of WL will get excluded
+            wl_list = self.getWorkloadList()
+            LOG.debug("\nWL lists are : {}\n".format(str(wl_list)))
+            wl_instance_id = []
+            for i in wl_list:
+                body = self.getWorkloadDetails(i)
+                LOG.debug("Body from WorkloadDetails:" + str(body))
+                for i in range(0, len(body['instances'])):
+                    wl_instance_id.append(body['instances'][i]['id'])
+                LOG.debug("\nWL instances are: " +str(wl_instance_id))
+
+            protected_vm_list = []
+            for vm in wl_instance_id:
+                if vm in vm_list:
+                    protected_vm_list.append(vm)
+            LOG.debug("\nProtected VMs are: " +str(protected_vm_list))
+
+            # Get tenant usage CLI command
+            get_tenant_usage = command_argument_string.get_tenant_usage
+            LOG.debug("Get tenant usage command: {}".format(get_tenant_usage))
+            rc = cli_parser.cli_returncode(get_tenant_usage)
+            if rc != 0:
+                reporting.add_test_step(
+                    "Execute get-tenants-usage command", tvaultconf.FAIL)
+                raise Exception("Command did not execute correctly")
+            else:
+                reporting.add_test_step(
+                    "Execute get-tenants-usage command", tvaultconf.PASS)
+                LOG.debug("Command executed correctly")
+            
+            # Compare total VMs hosted on a cloud vs total_vms showing in CLI output
+            out = cli_parser.cli_output(get_tenant_usage)
+            total_vm = cli_parser.cli_response_parser(out,'total_vms')
+            if (len(vm_list) == int(total_vm)):
+                reporting.add_test_step(
+                    "Verify total VMs on OpenStack and CLI ouput", tvaultconf.PASS)
+            else:
+                reporting.add_test_step(
+                    "Verify total VMs on OpenStack and CLI ouput", tvaultconf.FAIL)
```

Comment:
in this case, need to mark test script status as FAIL as well

Reply:
added